server/src/main/java/org/opensearch/index/translog/transfer/FileTransferTracker.java

```diff
@@ -59,6 +62,11 @@ void recordBytesForFiles(Set<TransferFileSnapshot> toUpload) {
             } catch (IOException ignored) {
                 bytesForTlogCkpFileToUpload.put(file.getName(), 0L);
             }
+            try {
+                bytesForTlogCkpFileToUpload.put(file.getCkpFileName(), file.getCkpFileContentLength());
```

Comment:
We could use a generation class `TranslogCheckpointSnapshot` for tracking the length and for other places wherever we can.

Reply:
Done